SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -188,6 +188,10 @@ udevadm trigger -y nitro_enclaves
 %{ne_include_dir}/*
 
 %changelog
+* Tue Feb 02 2021 Gabriel Bercaru <bercarug@amazon.com> - 1.0.10-0
+- Removed the %posttrans scriptlet and delegated the task of
+  re-performing resources initialization to a trigger script
+  which runs only then uninstalling v1.0 or the package
```

Comment:
Would the trigger be run also during uninstall (at all, not during package update)? Should not be needed then. Just clarify a bit the text here "... or the package".

Reply:
Yes, it is run at plain uninstall, but in order to happen, the user should have both this version (which uses this spec file version, for example `aws-nitro-enclaves-cli-1.0.10-0`) and the previous package version (`aws-nitro-enclaves-cli` v1.0). If this is possible than indeed the trigger script should be updated in order to perform only on package update.

Reply:
Thanks for info. Right, if that's needed, a check should be for the package action, if update or removal at all.

Reply:
I tested and saw that the package operation during the execution of the trigger script is `2`, so I added the corresponding check, to be on the safe side (even tough I couldn't get the trigger script to run when doing a simple package removal).